File: password_generator/java/Op1/Main.java
package password_generator.java.Op1;

public class Main {
    public static void main(String[] args) {
        Password myPassword = new Password(true, true, true);
        System.out.println(myPassword.generate(16));
        Password myPassword2 = new Password(false, false, false);
        System.out.println(myPassword2.generate());
        Password myPassword3 = new Password(false, false, true);
        System.out.println(myPassword3.generate(16));
    }
}

File: password_generator/py/op1/main.py
from Password import Password

password = Password(upper_leters=True, numbers=True, symbols=True)
print(password.generate(16))
myPassword2 = Password();
print(myPassword2.generate());
myPassword3 = Password(upper_leters=False, numbers=False, symbols=True);
print(myPassword3.generate(16));
myPassword3 = Password(upper_leters=True, numbers=False, symbols=True);
print(myPassword3.generate(16));